File: Gerontocracy.Web/src/gerontocracy/modules/party/models/politiker-overview.ts
export interface PolitikerOverview {
    id: number;
    externalId: number;
    name: string;
    wahlkreis: string;
    bundesland: string;
    parteiId?: number;
    parteiKurzzeichen: string;
    isInactive: boolean;
    reputation: number;
}

File: Gerontocracy.Core/BusinessObjects/Party/PolitikerOverview.cs
namespace Gerontocracy.Core.BusinessObjects.Party
{
    public class PolitikerOverview
    {
        public long Id { get; set; }
        public long ExternalId { get; set; }

        public string Name { get; set; }
        public string Wahlkreis { get; set; }
        public string Bundesland { get; set; }
        public bool IsInactive { get; set; }
        public int Reputation { get; set; }

        public long? ParteiId { get; set; }
        public string ParteiKurzzeichen { get; set; }
    }
}
